### ISSUE TITLE
Setting query parameters in mgconsole

### ISSUE THREAD

**Issue by g-despot** [memgraph/mgconsole#31]
It would be useful to have the option of setting parameter values in **mgconsole** directly.
For example:

```
:param name => 'John'
```

**Comment by gitbuda** [comment]
Yep, I had the same issue during `Code with Buda` -> https://youtu.be/tiSP255Ukx0?t=3330 :smile:
